test/tomcat/server.xml

```diff
@@ -24,10 +24,6 @@
             advertise="true" stickySession="true" stickySessionForce="false" stickySessionRemove="true" /> -->
   <Listener className="org.jboss.modcluster.container.catalina.standalone.ModClusterListener"
             connectorPort="8080"
-            advertise="false"
-            stickySession="true"
-            stickySessionForce="false"
-            stickySessionRemove="true"
```

Comment:
FYI - the stickySessionRemove is false by default. But clearly that's not tested anywhere here.